### ISSUE TITLE
Don't append the modpack's name to version names automatically

### ISSUE THREAD

**Issue by HyperSoop** [modrinth/knossos#1372]
### Is your suggested enhancement related to a problem? Please describe.

Not sure if this applies to other project types, but when I drag and drop a version of my modpack onto the versions page to upload it, the version's name starts with the modpack's name by default. I don't think there's any reason version names should contain the project's name.

### Describe the solution you'd like

Don't append the modpack's name to the version name when providing initial metadata.

**Comment by triphora** [comment]
It's a fairly standard approach; [Sodium does this](https://modrinth.com/mod/sodium/versions), as do all of the other top 5 mods on Modrinth.

**Comment by HyperSoop** [comment]
it's fine then, just imo it's redundant to include this information in version names

**Comment by Prospector** [comment]
I agree it should stop, it's a common, but bad practice. I stopped doing this for Mod Menu recently, it's way better: 

![Screenshot_20230917-001008.png](https://github.com/modrinth/knossos/assets/6166773/c70cc82d-26c0-4098-a9a0-7c8b9e05824c)

Version titles should just pertain to the version, the project name is redundant and unnecessary and conveys no helpful information

**Comment by Minenash** [comment]
Yeah I also stopped doing it. It's really bad, especially with long mod names
